src/style.css

```
@@ -1,3 +1,85 @@
+/* stylelint-disable block-no-empty */
+html {
+  box-sizing: border-box;
+}
+
 body {
+  box-sizing: border-box;
```

move `box-sizing: border-box;` to `*`. This rule is only allowed to use at this selector
```suggestion
}

body {
```